packages/api-client/src/components/EnvironmentSelector/EnvironmentSelector.vue

```diff
@@ -0,0 +1,102 @@
+<script setup lang="ts">
+import { useWorkspace } from '@/store/workspace'
+import {
+  ScalarButton,
+  ScalarDropdown,
+  ScalarDropdownItem,
+  ScalarIcon,
+} from '@scalar/components'
+import { computed } from 'vue'
+import { useRouter } from 'vue-router'
+
+const {
+  environments,
+  activeWorkspace,
+  workspaceMutators,
+  activeEnvironment,
+  isReadOnly,
+} = useWorkspace()
+
+const router = useRouter()
+
+const updateSelected = (uid: string) => {
+  workspaceMutators.edit(activeWorkspace.value.uid, 'activeEnvironmentId', uid)
+}
+
+const createNewEnvironment = () =>
+  router.push(`/workspace/${activeWorkspace.value.uid}/environment`)
+
+const envs = computed(() => {
+  return Object.values(environments).filter((env) => env.uid !== 'default')
+})
+</script>
+<template>
+  <div>
+    <ScalarDropdown>
+      <ScalarButton
+        class="font-normal h-auto justify-start py-1.5 px-1.5 text-c-1 hover:bg-b-3 text-c-2 w-fit"
+        fullWidth
+        variant="ghost">
+        <h2 class="font-medium m-0 text-sm flex gap-1.5 items-center">
+          {{ activeEnvironment?.name ?? 'No Environment' }}
```

Comment:
should we display the `Global Environment` name here to be consistant with what we display in the environment page?